src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayout.Layouting.Measure.cs

```diff
@@ -283,7 +283,7 @@ private void ApplyMinMaxValues(ref Size desiredSize)
 		{
 			var child = _calculatedChildren![i];
 
-			if (child.Role == AutoLayoutRole.Filled || child.IsVisible)
+			if (child.Role == AutoLayoutRole.Filled && child.IsVisible)
```

Comment:
Is this something we can add a runtime test for?

Reply:
That a good point, I will take a look

Reply:
That's required :-)